addons/resource.language.en_gb/resources/strings.po

```diff
@@ -6818,7 +6818,18 @@ msgctxt "#13516"
 msgid "Add art"
 msgstr ""
 
-#empty strings from id 13517 to 13549
+#: system/settings/settings.xml
+msgctxt "#13517"
+msgid "Threshold when atempo resampling kicks in"
```

Comment:
This is wrong in almost every sense :)

> Above this speed

threshold is no speed

> audio is resampled to preserve the pitch

it is the other way round. resampling does not preserve the pitch

> This has a minor performance impact

we don't have any measurements on how atempo filter compares to resampling. apart from pi both is done by CPU.